### PR TITLE
Create Package on AppVeyor

### PR DESCRIPTION
**Pull request by 304NotModified**
- Create and collect package on AppVeyor
- Prepare for V3 rc1 release
- Prepare for Deterministic Builds

follow up of #124 - will rebase when #124 is merged

### CODE REVIEW

**Comment by 304NotModified** [comment]
See https://ci.appveyor.com/project/yaxlib/yaxlib/build/job/d8ceie5i71qo76sg/artifacts

image

File: YAXLib/YAXLib.csproj [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
-    </ItemGroup>
-    
-    <ItemGroup>
-        <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
-            <_Parameter1>YAXLibTests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d7f87eefee568e19ff867da48567811d16853dbc045adcae7a9682012ad27a39dd24c9959db4e87965a589859f8a2cfc33c2d4997bf969dc4baa159ceefe1a85f45368d76918c764df03763848f50370e660da01bb4b2614fb67f8d0f7183c0c640b16c61d47628d8e4ba20f86dc6bab11b0528425efd05607c28ac576f92ca1</_Parameter1>
-        </AssemblyAttribute>
-    </ItemGroup>
-    
+
+  <PropertyGroup>
```
Comment (304NotModified) [inline YAXLib/YAXLib.csproj:3]: hint: 

image

**Comment by 304NotModified** [comment]
Next step after this, is publishing to NuGet

**Comment by 304NotModified** [comment]
rebased on master, ready for review

**Review by axunonb — COMMENTED** [review]
Sorry, never did that before, not possible give any feedback.

**Comment by 304NotModified** [comment]
@axunonb 

Well you can see that the package is correct on AppVeyor? 

1. Go to the build
2. Go to the windows build
3. Go to tab artifacts
4. Download and check the package?

**Review by axunonb — COMMENTED** [review]
Thanks for your hint. The packages are fine so for. Just 2 questions for understanding:
* I always thought, that `nupkg` still contain the `pdb`s and `xml`s. Looks like this is wrong, and `xml`s are not part of the packages at all (when using source link)?
* Although the .NetStandard2.0 dependency for `PlatformAbstraction` is removed, it still shows up as dependency in the packages?

Sorry for bothering.

**Comment by 304NotModified** [comment]
> * I always thought, that `nupkg` still contain the `pdb`s and `xml`s. Looks like this is wrong, and `xml`s are not part of the packages at all (when using source link)?

The pdb are now in the snupkg. The XML is indeed missing, this is #126

**Comment by 304NotModified** [comment]
close/reopen to trigger CI

**Review by axunonb — APPROVED** [review]

**Pull request merged**